x-pack/plugins/snapshot_restore/public/app/index.tsx

```diff
@@ -3,7 +3,7 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
+import { I18nContext } from 'ui/i18n';
```

Comment:
I think the Context is the one thing provided by core:

* https://github.com/elastic/kibana/blob/master/x-pack/plugins/remote_clusters/public/shim.ts#L22
* https://github.com/elastic/kibana/blob/master/x-pack/plugins/remote_clusters/public/plugin.js#L26